components/map.tsx

```diff
@@ -1,13 +1,16 @@
 import GoogleMapReact from "google-map-react";
+import React from "react";
+import Marker from "./Marker/Marker";
 
 export default function Page() {
   return (
     <div style={{ height: `100vh`, width: `100%` }}>
       <GoogleMapReact
         bootstrapURLKeys={{ key: process.env.NEXT_PUBLIC_GOOGLE_MAPS_KEY }}
         defaultCenter={{ lat: 30.2672, lng: -97.7393 }}
-        defaultZoom={10}
+        defaultZoom={16}
       />
+      <Marker lat={30.2672} lng={-97.7393} name="Crag Climbing" color="blue" />
```

Comment:
Instead of `name`, let's call this prop `text` so it'll be passed through to the return statement properly

Reply:
done